src/services/promoted_attribute_definition_interface.ts

```diff
@@ -0,0 +1,8 @@
+export interface DefinitionObject {
```

Comment:
I think we should move these things to a types.ts file or something similar.

Reply:
@alexpietsch , these might look a bit weird for now, but once we do the ESM conversion I would want to move them back in to the file they were first identified. That's why the files are named `name-of-an-existing-file_interface.ts`.